Wabbajack.App.Blazor/State/IStateContainer.cs

```diff
@@ -17,9 +17,18 @@ public interface IStateContainer
     IObservable<AbsolutePath> ModlistPathObservable { get; }
     AbsolutePath ModlistPath { get; set; }
     
+    IObservable<AbsolutePath> InstallPathObservable { get; }
+    AbsolutePath InstallPath { get; set; }
+    
+    IObservable<AbsolutePath> DownloadPathObservable { get; }
+    AbsolutePath DownloadPath { get; set; }
+    
     IObservable<ModList?> ModlistObservable { get; }
     ModList? Modlist { get; set; }
     
+    IObservable<string> ModlistImageObservable { get; }
+    string ModlistImage { get; set; }
```

Comment:
Should be nullable. Would require changes elsewhere and some `is not null` checks but it provides more safety.